specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2490,12 +2914,12 @@
           "type": "string",
           "description": "The display name of the bookmark"
         },
-        "lastUpdatedTimeUtc": {
+        "updated": {
```

Comment:
This is a breaking change, because its preview you don't have to introduce a new API version but worth communicating with partners.

Reply:
done